src/components/flag/code/templates/FlagComponentTemplate.jsx

```diff
@@ -3,21 +3,12 @@ var classNames = require('classnames');
 
 module.exports = function(props) {
   var classes = {
-    'ui-component-flag': true,
-    'flag-left': props.left,
-    'flag-right': props.right,
-    'flag-top': props.top,
-    'flag-bottom': props.bottom,
-    'flag-primary': props.primary,
-    'flag-secondary': props.secondary,
-    'flag-success': props.success,
-    'flag-info': props.info,
-    'flag-warning': props.warning,
-    'flag-danger': props.danger,
-    'flag-sm': props.small,
-    'flag-md': props.medium,
-    'flag-lg': props.large
+    'component-flag': true,
   };
+  if(props.size) classes[props.size] = true;
+  if(props.purpose) classes[props.purpose] = true
+  if(props.position) classes[props.position] = true
```

Comment:
@joestump 

This is a controversial one here, the dev team have always been accepting of one line if statements and they are everywhere in our projects. Having a look at the [eslint docs](http://eslint.org/docs/rules/curly#options), it looks as though options can be passed in to be more relaxed on the one liners but still enforce curly brackets on blocks with 2+ statements.

```javascript
curly: [2, "multi-line"]
```